Documentation/operations/upgrade.rst

```diff
@@ -355,6 +355,12 @@ Deprecated Options
 * The ``force-local-policy-eval-at-source`` option is deprecated and will be
   removed in 1.14.
 
+New Options
+~~~~~~~~~~~
+
+* ``cni-exclusive``: If set, this tells the agent to remove non-Cilium CNI configuration files
+* ``cni-log-file``: A file, relative to the host, where the CNI plugin can write logs.
+
```

Comment:
This is currently extending the v1.13 release notes. Can you push this into a new section for v1.14? (We should probably delete all the current 1.13 release notes from the master branch at some point, but I don't want to do that too early before we've finalized the release notes so maybe it's better to just retain the v1.13 release notes for now and just move these to a brand new v1.14 section).

Reply:
oh oops, good catch.

Do you care if I just document this later? I will have even more fun options coming soon.

Reply:
@joestringer I removed the upgrade notes for now. I'm going to be doing a lot more CNI work in the coming weeks, so I promise it'll be documented :-)